vuepress/docs/next/tutorials/solution/strapi.md

```diff
@@ -0,0 +1,183 @@
+
+The configuration of a Strapi instance and content are achieved using the Strapi packaged business capability (PBC) available on the Entando Cloud Hub. This tutorial covers:
+
+1. [Prerequisites](#prerequisites)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+4. [Next Steps](#next-steps)
+
+## Prerequisites
+
+- [A working instance of Entando](../../../docs/getting-started/)
+- Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Installation
+
+Entando's Strapi implementation is available from the Entando Cloud Hub via 3 bundles, which must be installed in the Local Hub of the App Builder.
+
+1. Within your Entando instance, create a persistent volume claim with the following properties: 
+   - Name: `pn-fe9131bb-ca5e5232-entando-strapi-server-pvc`
+   - Storage size: 2GB
+
+2. Using the ent CLI, apply the Custom Resource Definitions for the Strapi bundles:
+
+```
+ent ecr deploy --repo=https://github.com/Entando-Hub/entando-strapi-bundle.git
+ent ecr deploy --repo=https://github.com/Entando-Hub/entando-strapi-config-bundle.git
+ent ecr deploy --repo=https://github.com/Entando-Hub/entando-strapi-widgets-bundle.git
+```
+
+3. Install the bundles into your Local Hub:
+
+```
+ent ecr install entando-strapi-bundle
+ent ecr install entando-strapi-config-bundle
+ent ecr install entando-strapi-widgets-bundle
+```
+
+### Roles
+
+#### Entando Strapi Config Application
+
+All REST APIs are public. No Keycloak role is required.
+
+#### Entando Strapi Content Template Application
+
+All REST APIs are private and require the admin role. To add Keycloak role mapping for the `entando-strapi-config` and `entando-strapi-templates` clients:
+
+1. [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin
+2. From the left menu, select `Users`
+3. Click on the ID associated with the admin username
+3. Click on the tab `Role Mappings`
+4. From the `Client Roles` drop-down, select the role ending in "strapi-config-server"
+   - Select all `Available Roles` and add them to `Client Default Roles`
+5. From the `Client Roles` drop-down, select the role ending in "strapi-template-server"
+   - Select all `Available Roles` and add them to `Client Default Roles`
+
+![strapi-keycloak-settings.png](./strapi-images/strapi-keycloak-settings.png)
+
+#### Entando Strapi Application
+
+Access to Strapi admin APIs requires the the Super Admin user role and same username on record with the App Builder. The existing Entando Keycloak token is used for authentication.
+
+### Registration
+
+Strapi registration is available following bundle installation. To log in to Strapi:
+
+1. Open a browser tab and enter your App Builder base URL followed by `/entando-strapi/admin/`, e.g. `http://hubdev.okd-entando.org/entando-strapi/admin/`
```

Comment:
Let's not use a development server as an example here. I think you can just change this to https://YOUR-SERVER-URL/entando-strapi/admin/